src/containers/DAO/pages/DAOPage/Proposals/ProposalDetail/ProposalAttributes/ProposalAttributes.tsx

```diff
@@ -95,6 +96,20 @@ export const ProposalAttributes: React.FC<ProposalAttributesProps> = ({
 
 		if (isFromSnapshotWithMultipleChoices(proposal)) {
 			parsedAttributes = parsedAttributes.slice(1);
+		} else {
+			parsedAttributes.splice(2, 0, { label: 'Type', value: 'Funding' });
+			parsedAttributes.splice(3, 0, {
+				label: 'Amount',
+				value:
+					formatTotalAmountOfTokenMetadata(proposal.metadata)?.toString() ||
+					'-',
+			});
```

Comment:
nit: is there a reason we're using `splice` here instead of just `parsedAttributes[2] = { ... }` or `parsedAttributes.push`?